src/lib/core/overlay/overlay-directives.spec.ts

```diff
@@ -29,16 +29,25 @@ describe('Overlay directives', () => {
     fixture.detectChanges();
   });
 
+  /** Returns the current open overlay pane element. */
+  function getPaneElement() {
```

Comment:
It may be better to use `querySelector` here, otherwise the test will break if we changed the DOM order.